templates/data-sources.md.tmpl

```diff
@@ -20,6 +20,8 @@ subcategory: "{{- $group := index (split .Name "_") 1 -}}
   Threat Defense
 {{- else if eq $group "federation" -}}
   IPAM Federation"
+{{- else if eq $group "cloud" -}}
+  Cloud Discovery Providers
```

Comment:
this might not be true always, there is also a "cloud_forwarder" api that might come up later

Reply:
This is would require changing the $group variable as it splits the name using the underscore and takes in "1" element written . 
https://github.com/infobloxopen/terraform-provider-bloxone/blob/b5132fea41e6e2addfe90cdcf341ffa9732b0f68/templates/resources.md.tmpl#L4

Do you recommend writing a new resource file ?

Reply:
yes, a different template might be good here, since the examples also need to be separated.

Reply:
Added a new template.

Reply:
Added the template for data source